3.processing_features/scripts/0.merge_sc_cytotable.py

```diff
@@ -74,7 +74,7 @@
         ).resolve(strict=True)),
         "dest_path": str(pathlib.Path(f"{output_dir}/{name}.parquet")),
     }
-    for name in plate_names if name=="Plate_5" # focus on plate 5
+    for name in plate_names if name in ["Plate_3", "Plate_3_prime", "Plate_4"]  # focus on Plate_3, Plate_3_prime, and Plate_4
```

Comment:
Could also make this more dynamic to accept n plates as input, although not necessary

Reply:
I guess you could use the plate_info yaml file for this

Reply:
I don't think n will work here since we need specific plates and just a number might be harder to get the right plates. This is a simple way for me to make sure that at the beginning of the preprocessing it will use the right plates.